qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -186,6 +190,18 @@ def _validate(self, qobj, backend_options, noise_model):
         """Validate the qobj, backend_options, noise_model for the backend"""
         pass
 
+    def _save_job(self, result, backend_options):
+        if (backend_options is not None) and ("path_to_save" in backend_options):
+            path_to_save = backend_options["path_to_save"]
+            if not path_to_save.endswith("/"):
+                path_to_save += "/"
+            filename = path_to_save + "_".join(
```

Comment:
Let's change the name so the date is in first place, this way it's easier to list them with `ls -la`.
Also, a more readable date/time format is encouraged so we can have more info about the job by just looking at the file name:
```
ss-mm-hh-dd-MM-yy-name-jobid.job
```

Reply:
Sure!